cvat-core/src/axios-tus.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import Axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import * as tus from 'tus-js-client';
+
+class AxiosHttpResponse implements tus.HttpResponse {
+    readonly #axiosResponse: AxiosResponse;
+
+    constructor(axiosResponse: AxiosResponse) {
+        this.#axiosResponse = axiosResponse;
+    }
+
+    getStatus(): number { return this.#axiosResponse.status; }
```

Comment:
Please, follow the general codestyle, e.g.

```
getStatus(): number {
    return this.#axiosResponse.status;
}
```

Reply:
Okay, reformatted.